### PR TITLE
AZ-376: Produce aleph-runtime artifact

### PR DESCRIPTION
**Pull request by pmikolajczyk41**

### CODE REVIEW

File: .github/workflows/e2e-tests-main-devnet.yml [hunk 1]
```diff
@@ -80,6 +78,11 @@ jobs:
           export RUSTC_WRAPPER=$HOME/.local/bin/sccache
           cargo build --release
 
+      - name: Build runtime
+        run: |
+          export RUSTC_WRAPPER=$HOME/.local/bin/sccache
+          cargo build --release -p aleph-runtime
```
Comment (timorl) [inline .github/workflows/e2e-tests-main-devnet.yml:84]: Huh, this really builds the wasm runtime? Reading the `Cargo.toml` I would not have expected that – am I missing something?

**Review by timorl — APPROVED** [review]
Looks reasonable, one question but if I'm being silly just resolve it.

File: .github/workflows/e2e-tests-main-devnet.yml (hunk 1 above)
Comment (pmikolajczyk41) [reply .github/workflows/e2e-tests-main-devnet.yml:84]: Hmm...
by `-p aleph-runtime` we tell cargo to build only the package `aleph-runtime` and I guess is essentially the runtime, isn't it?

here you have the artifacts, including the output of this command: https://github.com/Cardinal-Cryptography/aleph-node/actions/runs/1583175148, seems reasonable in size

Comment (pmikolajczyk41) [reply .github/workflows/e2e-tests-main-devnet.yml:84]: oh, I get it now; please look at `build.rs`

**Pull request merged**